certified-connectors/Emigo/Readme.md

```diff
@@ -0,0 +1,48 @@
+#Title
```

Comment:
@tomaszsagra please follow our templates here for the readme file. you can find an example [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) and for raw view for code [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md)). please include all required sections.